tests/Stub/StubCommand.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Console\Tests\Stub;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Yiisoft\Yii\Console\ExitCode;
+use Yiisoft\Yii\Console\Application;
+
+class StubCommand extends Command
+{
+    protected static $defaultName = 'stub';
+
+    public function configure(): void
+    {
+        $this
+            ->setDescription('Stub command tests');
+    }
+
+    public function __construct(Application $application)
+    {
+        $this->application = $application;
```

Comment:
I think `private $application` should be defined since parent one can not be accessed.

Reply:
done.